### PR TITLE
Design Project: BUILDING WITH RESPONSIVE DESIGN

### PR DESCRIPTION
**Pull request by webmarkyn**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,804 @@
...
+}
+
+.side-hero-each small {
+  color: rgb(189, 188, 188);
+  font-size: 12px;
+}
+
+.side-hero-1 {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)), url(images/hero/sidehero1.png);
+  background-position: center;
```
Comment (SunnyAsar) [inline style.css]: the link to your image assets doesn't match what's in the project. so its all blank:
Screenshot 2019-09-26 at 1 10 10 PM

File: style.css [hunk 2]
```diff
@@ -0,0 +1,804 @@
...
+  .funds {
+    flex: 0 1 33.33%;
+  }
+
+  .funds:last-of-type {
+    display: none;
+  }
+}
+
+@media only screen and (max-width: 655px) {
```
Comment (SunnyAsar) [inline style.css]: we need some media queries to handle the navigation especially, we should be displaying a hamburger icon for mobile

File: index.html [hunk 3]
```diff
@@ -0,0 +1,453 @@
...
+        <nav>
+
+            <div class="nav-icon"></div>
+
+            <a href="#" class="nav-links">Latest</a>
+            <a href="#" class="nav-links">Hard Fork</a>
+            <a href="#" class="nav-links">Plugged</a>
+            <a href="#" class="nav-links">The Next Police</a>
+            <a href="#" class="nav-links">Fundamentals</a>
+            <input class="search-bar" type="text" placeholder="Search">
```
Comment (SunnyAsar) [inline index.html]: there is supposed to be  a sub nav here

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hello @webmarkyn  and @rkyankya 👋 
well done guys 😃 , we are well on our way to a wonderful project, lets just hit all the points and make it look exactly like [thenextweb.com](https://thenextweb.com/)
### lets add these:
* [ ] It contains sub-navigation row.
* [ ] It contains search icon at the rightmost side of sub-navigation row.

###### Medium version changes
* [ ] TWN logo appears at the leftmost part. (The logo is not always there)
###### Small version changes
* [ ] Navigation and sub-navigation row changes to a hamburger (bars) icon at the rightmost part (next to search icon).

##### Footer
* [ ] It contains social media icons vertically and horizontally aligned.
* [ ] It contains TNW logo with a copyright paragraph.


#### Finally
 Please look at the original site and the nav behaviour especially and make update accordingly.


**Make another review request when the updates are in place** Cheers!

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,505 @@
...
+            <h1>Supercharge your Echo device with these essential Alexa skills</h1>
+            <p>by <a href="#">Tristan Greene</a> 18 hours ago 134</p>
+          </article>
+      </header>
+
+    </div>
+
+  </section>
+
+  <section class="latest-news">
```
Comment (Torres-ssf) [inline index.html]: **[REQUIRED]**
- [ ] We need to fix the position of the latest news heading.
Screenshot from 2019-09-27 07-38-20

File: index.html [hunk 5]
```diff
@@ -0,0 +1,505 @@
...
+    </header>
+
+    <div class="grid news-grid grid-4-3-1">
+
+        <header class="latest-story" id="latest-1">
+            <a class="latest-image-wrapper" href="#"><img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2017/12/guy-2609815_1920-405x270.jpg" alt="Image"></a>
+            <a href="#"><h3 class="headline">The first SMS was sent 25 years ago. Here's why it's still relevant.</h3></a>
+            <a class="byline" href="#">Matthew Hughes - 17 hours ago</a>
+        </header>
+        <header class="latest-story" id="latest-2">
```
Comment (Torres-ssf) [inline index.html]: [TIP]
Why did you use the header tag as a container for each latest news item here? :thinking: . As it said in the [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header), a `header` element is intended to usually contain the surrounding section's heading. A better approach is to use `article` tags instead for each item.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,505 @@
...
+              <li><a>How texting changed language</a></li>
+              <li><a>The first SMS was sent 25 years ago. Here's why it's still relevant.</a></li>
+            </ul>
+          </div>
+      </article>
+
+    </div>
+
+  </section>
+
```
Comment (Torres-ssf) [inline index.html]: **[REQUIRED]**
- [ ] I notice that you forget to implement the Latest deals section. We need to fix this. Remember that we also need to make it resposive.
Screenshot from 2019-09-27 07-35-31

File: README.md [hunk 7]
```diff
@@ -1 +1,25 @@
...
+Microverse project
+
+This project consists of building an HTML document that matches the appearance of https://thenextweb.com/ web page.
+
+You can find the original project specification at:
+ <https://www.theodinproject.com/courses/html5-and-css3/lessons>  
+<https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-responsive-design>
+
+Author:
+This project was executed by Mark Baidebura and Kyankya Raymond students of Microverse.
```
Comment (Torres-ssf) [inline README.md:15]: **[REQUIRED]**
Great job creating a descriptive readme file. But it's also a best practice to add some styles to it. By doing that you are making it more easy to read and understand. And it also gives a professional look. Check out this [template](https://github.com/othneildrew/Best-README-Template) about creating awesome readme files.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Hi there team :hand: :smiley: 

Great work implementing the suggested changes. :clap: 
Your page looks really nice and you did a great job making it responsive :tada: 

**But we still have some change to make**
Notice that you need to do all REQUIRED changes in order to get this PR approved.

**[REQUIRED]**
- [ ]  There are some errors in your HTML code. Check them using this HTML [validator](https://validator.w3.org/) . Ignore the warnings.

Please kindly make these changes and submit for code review again. Happy coding :computer:

**Comment by ebukaume** [comment]
### Hi @webmarkyn  and @rkyankya  :hand: 

You have really done some great jobs here. But you missed some important requirements. :wink: 

1. Please always validate your HTML code [here](https://validator.w3.org) to save you time because we will always find any validation errors. Please fix the duplicate ID error [here](https://github.com/webmarkyn/Responsive-Design/blob/Design/index.html#L357) and [here](https://github.com/webmarkyn/Responsive-Design/blob/Design/index.html#L543)

2. Also, you can do better with your README file as @Torres-ssf pointed out earlier. You can look at this [nice example](https://github.com/othneildrew/Best-README-Template) and make your README more professional. Guys this is always the first thing visitors to your repo see, so it's important we keep it as professional as possible. :smile: 

Please submit another code review request after you have made these changes. :v:

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Hello there team :hand: 

Great job validating your page and remove the errors :+1: 

But I noticed that you removed the most Latest Funding section from its place, duplicated it, and placed it in the Latest Deals section. I'm sorry but Microverse guidelines for this project require that the cloned page must have all sections from the original page. We TSE are not allowed to approve any projects that don't met the guidelines :cry:.

**[REQUIRED]**
- [ ] Let's bring the Latest funding back to its place and create Latest deals section.

Please kindly make this change and submit for code review again. 
Happy coding :computer:

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Awesome job :+1: team
You are basically one step from completing this project. But before that, please make sure ```stickler``` checks pass.
Do that and send for another review. You can also slack me @whiz25 if you have a challenge doing that.

**Comment by whiz25** [comment]
Don't forget to remove the horizontal scrollbar from your page.

**Review by stratospherique — APPROVED** [review]
### Great work implementing the suggested changes :clap: . Your project is approved :heavy_check_mark: and you may merge to master :+1: :+1:

**Pull request merged**